### PR TITLE
Add missing grants to reports

### PR DESCRIPTION
**Pull request by jasalisbury**
## Description of change

Adds script that adds missing grants to legacy reports. The IDs of the reports were found in production by searching for their display ID.

## How to test

 * Run tests
 * Run the script locally, note all the "missing grant" messages
 * Double check my mapping of display id to database id. Also verify CSV has correct database id.

|Display|Id|
|-|-|
|R02-AR-000726|312|
|R02-AR-000722|1020|
|R02-AR-000716|924|
|R02-AR-000671|925|
|R02-AR-000663|365|
|R02-AR-000650|346|

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-75

### Every PR

- [x] Meets issue criteria
- [x] JIRA ticket status updated
- [x] Code is meaningfully tested
- [n/a] Meets accessibility standards (WCAG 2.1 Levels A, AA)
- [n/a] API Documentation updated
- [n/a] Boundary diagram updated
- [n/a] Logical Data Model updated
- [n/a] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions

### Production Deploy

- [ ] Staging smoke test completed

### After merge/deploy

- [ ] Update JIRA ticket status

### CODE REVIEW

File: src/tools/addMissingGrantsToReports.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    const activityReportId = row[PRIMARY_COLUMN];
+    const reportDisplayId = row[REPORT_ID_COLUMN];
+    const grantee = row[GRANTEE_NAME_COLUMN];
+
+    const grantNumber = grantee.split('|')[1].trim();
+    const grantId = grantsByNumber[grantNumber];
+
+    if (!grantId) {
+      // eslint-disable-next-line no-console
+      console.warn(`Unable to find grant ${grantee} (number ${grantNumber}) for report ${reportDisplayId} (id ${activityReportId})`);
```
Comment (thewatermethod) [inline src/tools/addMissingGrantsToReports.js:27]: should we switch this to use logger.warn?

**Review by thewatermethod — APPROVED** [review]
Looks good to me. Checked the csv vs the table, that lines up.

**Review by kryswisnaskas — CHANGES_REQUESTED** [review]
Looks good so far with one change requested. I am still reviewing, but it looks like the script imports the data successfully to the db 👍 

Can we also reconcile the display on the front end - either update the `imported `field or show the new list of grantees using a different source. Right now, they are being pulled from the legacy `imported `column.

**Comment by jasalisbury** [comment]
> Looks good so far with one change requested. I am still reviewing, but it looks like the script imports the data successfully to the db 
> 
> Can we also reconcile the display on the front end - either update the `imported `field or show the new list of grantees using a different source. Right now, they are being pulled from the legacy `imported `column.

Good catch! I've updated to adding the grants to the `imported` field. I removed adding ActivityRecipients so all legacy reports are the same (none have any ActivityRecipients). Hopefully that will make updating legacy reports adding recipients easier!

**Review by kryswisnaskas — APPROVED** [review]
Looks good! 👍

**Pull request merged**